src/l402/fetch.ts

```diff
@@ -1,11 +1,14 @@
 import MemoryStorage from "../utils/Storage";
 import NoStorage from "../utils/Storage";
 import { WebLNProvider } from '@webbtc/webln-types';
+import { parseL402 } from "./parse";
 
 export * as Storage from "../utils/Storage";
 const memoryStorage = new MemoryStorage();
 
-export const fetchWithLsat = async (url: string, fetchArgs: Record<string, any>, options: Record<string, any>) => {
+const HEADER_KEY = "LSAT"; // we have to update this to L402 at some point
```

Comment:
doesn't this have to be updated now to match the spec? if there are some backends that still support the old version, should HEADER_KEY be passed as an argument, but defaault to `"L402"`?

Reply:
I guess most APIs currently use LSAT
and the blip says APIs should be backwards compatible. so imo it is safer to use LSAT here and some time in the future change it to L402